examples/resources/nios_dns_sharedrecord_cname/resource.tf

```
@@ -0,0 +1,27 @@
+// Create a Shared Record Group (Required as Parent)
+resource "nios_dns_sharedrecordgroup" "parent_sharedrecord_group" {
+  name = "example-sharedrecordgroup"
+}
+
+// Create a Shared CNAME Record with Basic Fields
+resource "nios_dns_sharedrecord_cname" "sharedrecord_cname_record_with_basic_fields" {
```

```suggestion
resource "nios_dns_sharedrecord_cname" "sharedrecord_cname_with_basic_fields" {
```
Please check this for other examples too